specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```
@@ -444,7 +444,7 @@
     "/entities/linking": {
       "post": {
         "summary": "Linked entities from a well-known knowledge base",
```

The hyphen from the summary, as it breaks APIM.

```suggestion
        "summary": "Linked entities from a well known knowledge base",
```

---

updated all occurences of this